python/shark_turbine/transforms/quantization/mm_group_quant.py

```diff
@@ -145,27 +150,31 @@ def run(self):
 
     def rewrite(self, mr: TransposedMMResult):
         none_to_q = lambda x: "?" if x is None else x
-        inline_module_asm = GROUP_MATMUL_TEMPLATE.format(
-            param_name=mr.param_name,
-            lowp_type="i4",
-            m=none_to_q(mr.m),
-            n=none_to_q(mr.n),
-            k=none_to_q(mr.k),
-            n_div=mr.n // 2,
-            group0=mr.n // self.group_size,
-            group1=self.group_size,
-            element_type=mr.element_type,
-        )
+        # TODO (ian): make generalizable and not specific for brevitas
+        if "lm_head.weight" not in mr.param_name:
+            inline_module_asm = GROUP_MATMUL_TEMPLATE.format(
+                param_name=mr.param_name[8:],
```

Comment:
Can you add a brief comment on why we're slicing `[8:]`?

Reply:
Adding a todo comment to make clear for now but the param_name's we are getting here from the model are all prepended with "_params.". To match the safetensor format I'm skipping that portion of the param name. It's ugly and I intend to address that when I add support with the RenameParametersPass to be able to do the mapping on the newly created globals in this pass.